packages/eclipse/src/styles/globals.css

```
@@ -238,6 +238,13 @@
     --color-stroke-sky: #0284c7;
     --color-stroke-violet: #7c3aed;
     --color-stroke-yellow: #ca8a04;
+
+    --chart-1: var(--color-foreground-pink);
+    --chart-2: var(--color-foreground-fuchsia);
+    --chart-3: var(--color-foreground-lime);
+    --chart-4: var(--color-foreground-sky);
+    --chart-5: var(--color-foreground-purple);
+    --chart-5: var(--color-foreground-neutral-weak);
```

Stylelint (17.3.0)

[error] Unexpected duplicate "--chart-5" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)